vendors/mediatek/manifest.cmake

```diff
@@ -1,6 +1,6 @@
 set(
     AFR_MANIFEST_SUPPORTED_BOARDS
-    mt7697hx-dev-kit
+    ""
```

Comment:
Mediatek is also supported according to Cobus

Reply:
I haven;t seen the CMakeFiles

Reply:
If you're on latest release-1.5, it's in `vendors/mediatek/boards/mt7697hx-dev-kit/`